types/v56/openapi.go

```diff
@@ -88,17 +88,17 @@ type ExternalNetworkV2 struct {
 	NetworkBackings ExternalNetworkV2Backings `json:"networkBackings"`
 }
 
-// ExternalNetworkV2IPRange defines allocated IP pools for a subnet in external network
-type ExternalNetworkV2IPRange struct {
+// OpenApiIPRangeValues defines allocated IP pools for a subnet in external network
+type OpenApiIPRangeValues struct {
```

Comment:
This is related to flipping type with alias as describe in https://github.com/vmware/go-vcloud-director/pull/532/files#r1044336377